### DISCUSSION TITLE
Use these system for add a section translations in markdown files

### DISCUSSION THREAD

**Discussion by Magicianred** [ptkdev/all-shields-cli#12] [Ideas]
I'll looking for a system for add a new languages in Markdown files, and I think that this will be the right :)

I try to explain, when I add a new language to my repository, I need to add any file markdown for the new language and add in any existing files the new link to the translation.

We can use your system (or create a new repo) for make it

We configure in a file the available languages, something to manage the different case people use translation (in separate folders or add a lang code to the file in the same folder, etc) and add the start comment and end comment in the README where it's place the language link navigation.

So there is no need to changes all the files anytime you add a language 

What do you think about it?

**Comment by ptkdev** [comment]
all-shields-cli it wasn't made for this purpose, but you can fork it and make a version that fits that

**Reply by Magicianred** [reply]
I'll do it, thanks :)
